package.json

```diff
@@ -25,6 +26,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.4.1",
     "react-select": "^3.1.0",
+    "react-select-country-list": "^2.1.2",
```

Comment:
Let's not keep this as a dependency. Only the essentials used in every single app should be in this repo.

Reply:
Actually these types of data regarding the list of counties usually come from a BE service and we fetch them once while bootstrapping the application and store them in redux store. Not sure why we need the list of countries.

Reply:
But if we want to have the list of countries in FE then, its usually saved it in a JSON and then the file is lazily loaded. (as the content can be upto 4 to 5KB in size)

Reply:
> Let's not keep this as a dependency. Only the essentials used in every single app should be in this repo.

Okay dai I've removed this package.

Reply:
> Actually these types of data regarding the list of counties usually come from a BE service and we fetch them once while bootstrapping the application and store them in redux store. Not sure why we need the list of countries.

Thanks dai Got it